CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- `into_base_16_string_no_padding` function to the test helpers to accommodate to Starknet Foundry output where addresses and selectors don't have trailing zeros after `0x` (#1137)
```

Comment:
```suggestion
- `into_base_16_string_no_padding` function to the test helpers (#1137)
```

Reply:
I can remove this, but don't you think is better to be more explicit in the changelog for people to understand the reason behind changes when debugging, etc.?

Reply:
Since the function could be used for other purposes, and we usually don't add this type of context, I vote we keep it as concise as possible.